pkg/appconfig/config.go

```diff
@@ -222,4 +222,5 @@ const (
 const OKTAAPIURL = "OKTA_API_URL"
 
 // OKTAAPIToken is the key for the Okta API token
-const OKTAAPIToken = "OKTA_API_TOKEN" // #nosec G101 false positive - not the actual API key itself
+// #nosec G101 false positive - not the actual API key itself
```

Comment:
this `nosec` comment was ignored when it was on the same line

🤷 it should have worked, based on the docs, but it works here